bindings/wasm/examples-account/src/node.ts

```diff
@@ -34,8 +34,8 @@ async function main() {
             return await unchecked();
         case "multiple_identities":
             return await multipleIdentities();
-        case "storage_test_suite":
-            return await storageTestSuite(async () => new MemStore())
+        case "custom_storage":
+            return await storageTestSuite()
```

Comment:
Rename the file `memory_storage` to `custom_storage` (or whatever name is decided on) and add it to the README.md list of examples.